htdocs/comm/action/class/actioncomm.class.php

```diff
@@ -2515,12 +2515,13 @@ public function sendEmailsReminder()
 	 * @param int		$percent	The new percent value for the event
 	 * @return int					1 when update of the event was suscessfull, otherwise -1
 	 */
-	public function updatePercent($id, $percent)
+	public function updatePercent($id, $percent, $usermodid)
```

Comment:
For compatibility, add  $usermodid = 0 into function declaration.
Also, add a line into the doc comment.

Reply:
Updated. Thank you